lib/models/hcktest_config.rb

```diff
@@ -20,6 +34,7 @@ class HCKTestConfig < T::Struct
       const :playlists_path, String
       const :filters_path, String
       const :tests_config, T.nilable(T::Array[TestConfig])
+      const :test_parameters, T.nilable(T::Array[TestParameter])
```

Comment:
I guess you should specify `default: []` instead of making it nilable. `Tests#test_parameters` will raise an error if this is `nil`.